lib/registries/addon/components/registration-form-view/template.hbs

```diff
@@ -1,14 +1,61 @@
+<div data-analytics-scope='Form navigation menu' local-class='NavigationMenu {{if (not this.showNav) 'Collapsed'}}'>
+    <div local-class='Header' data-test-anchor-nav-title>
```

Comment:
Maybe move the button outside this "Header" -- it sits over top of the first line of the form:
![screen shot 2019-02-27 at 12 37 06](https://user-images.githubusercontent.com/6776190/53510895-09291d80-3a8d-11e9-842d-96837df23706.png)